From 15e03f55853f26ac282edc8980d8d20b5547196e Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Tue, 16 Jan 2024 08:54:24 +0100
Subject: [PATCH 0875/1141] ARM: config: add display connector in multiv7
 addons fragment

Enable the display connector driver (link to HDMI).

Change-Id: I61fa1fef237445976e7a183dd9895d932f5f66ef
Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/351448
---
 arch/arm/configs/fragment-02-multiv7_addons.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/fragment-02-multiv7_addons.config b/arch/arm/configs/fragment-02-multiv7_addons.config
index 338cf511cc40..a3abc68b4469 100644
--- a/arch/arm/configs/fragment-02-multiv7_addons.config
+++ b/arch/arm/configs/fragment-02-multiv7_addons.config
@@ -19,6 +19,7 @@ CONFIG_DEBUG_INFO_DWARF_TOOLCHAIN_DEFAULT=y
 CONFIG_DEBUG_PREEMPT=y
 CONFIG_DEBUG_SECTION_MISMATCH=y
 CONFIG_DNS_RESOLVER=y
+CONFIG_DRM_DISPLAY_CONNECTOR=y
 CONFIG_DRM_I2C_ADV7511=y
 CONFIG_DRM_PANEL_ORISETECH_OTM8009A=y
 CONFIG_DRM_PANEL_RAYDIUM_RM68200=y
-- 
2.39.2

